src/models/tests/m_identity_tests.erl

```diff
@@ -42,7 +42,10 @@ check_password_no_user_test() ->
     ?assertEqual({error, nouser}, m_identity:check_username_pw("mr_z", "does-not-exist", C)),
     ok.
 
-check_username_password_test() ->
+check_username_password_test_() ->
+    {timeout, 20, fun() -> check_username_password() end}.
```

Comment:
Yes, I guess the db can be a bit slow on Travis. The default timeout is 5 seconds, which gave random timeout failures.